src/app/app-routing.module.ts

```diff
@@ -19,7 +19,7 @@ export const routes: Routes = [
   imports: [
     RouterModule.forRoot(routes, {
       preloadingStrategy: PreloadAllModules,
-      useHash: true
+      useHash: false
```

Comment:
false is default here, so this could be removed

Reply:
Because this 'false' value is mandatory here (or SSR won't work), I would prefer to set it explicitly.

For example, in a few years, a 'git blame' can give you the reason (commit) to set it as false.